website/docs/r/cloudwatch_event_rule.html.markdown

```diff
@@ -33,7 +33,23 @@ resource "aws_cloudwatch_event_target" "sns" {
 }
 
 resource "aws_sns_topic" "aws_logins" {
-  name = "aws-console-logins"
+  name   = "aws-console-logins"
+  policy = "${data.aws_iam_policy_document.sns_topic_policy.json}"
+}
+
+data "aws_iam_policy_document" "sns_topic_policy" {
+
+  statement {
+    effect  = "Allow"
+    actions = ["SNS:Publish"]
+
+    principals {
+      type        = "Service"
+      identifiers = ["events.amazonaws.com"]
+    }
+
+    resources = ["${aws_sns_topic.autoscaling_events.arn}"]
```

Comment:
I believe this is supposed to be `"${aws_sns_topic.aws_logins.arn}"`, which would create a self-reference issue. The policy will likely need to be applied as a separate `aws_sns_topic_policy` resource. 👍